### ISSUE TITLE
[eng] SC | Update private medical records section to more accurately represent 4142 and 4142a 

### ISSUE THREAD

**Issue by eileen-coforma** [department-of-veterans-affairs/va.gov-team#83774]
## Value Statement

**_As a_** Veteran applying for Supplemental Claim

**_I want to_** understand that by submitting private medical records, I am also filing out the 4142 and 4142a

**_So that_** I know that the online process includes these forms

---

## Background Context

Kayce pointed out that the way we currently include the 4142 and 4142a inline within the form is not compliant, and should not have passed OMB OIRA approval. By including the 4142 and 4142a within 0995 without identifying them as separate forms could increase respondent burden time. We need to clarify that the sections which include the 4142 and 4142a are separate from the 0995 form, even if the Veteran can fill it out at the same time.

## Outcome, Success Measure, KPI(S), and Tracking Link

- It would be helpful to get a baseline measure of Veterans submitting private medical records now VS after we update the form language.

## Acceptance Criteria

- [ ] Private medical records screens are updated to say they are part of 4142 and 4142a
   - [ ] ~https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/request-private-medical-records~
   - [x] https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/private-medical-records-authorization
   - [x] https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/private-medical-records
   - [x] https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/add-private-record-limitations
- [x] Veterans are aware of the OMB form number and respondent burden times, and can choose to fill it out separately. (is the omb form number and respondent burden necessary?)
- [ ] Design is complete and reviewed by enablement

## Designs and Build Notes

- [FIGMA](https://www.figma.com/design/2LGebZcUuu5Iqh4QLPII6A/Supplemental-Claims-(VA-0995)?node-id=919-33426&m=dev)

### Enablement team (if needed)
@jstrothman 

## Out of scope

- Other SC form updates

## Open questions

- What is necessary for an online form to be legally compliant? Does the OMB and respondent burden need to be surfaced before the Veteran chooses to enter the form?

## Tasks

- [ ] Update private medical records page to show that by selecting yes, they are optionally filling out the 4142 and 4142a forms
- [ ] Update private medical records pages to reflect the addition of the inline addition of 4142 and 4142a forms in the header
- [ ] Update authorization error message to let Veterans know they can optionally fill out the 4142 and 4142a forms later

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Sample data provided where appropriate
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Reviewed and approved by product and/or design

**Comment by jstrothman** [comment]
Regarding the open question in the main issue description:
>What is necessary for an online form to be legally compliant? Does the OMB and respondent burden need to be surfaced before the Veteran chooses to enter the form?

[VADS OMB info component page](https://design.va.gov/components/omb-info#when-to-use-omb-info) has useful quotes re the requirement to display it and re the placement. 

In Figma, I'm recommending (to be reviewed with CAIA) we take an approach of:

1. Adjust the sub-header (form number line) throughout the form to indicate it's VA Form 20-0995 with optional VA Forms 21-4142 and 21-4142a
2. On the intro page bottom, show OMB info for both forms (with heading for each)
3. Keep the url structure, i.e. it always has `file-supplemental-claim-form-20-0995` even on the 4142-related pages
4. Explanatory text as you suggested, @eileen-coforma, on the initial question to explain that by answering yes, we'll ask you questions from the 4142+4142a and submit it along with your Supplemental Claim (which we've been doing but not explicitly saying on this page)

**Comment by shaunburdick** [comment]
@skylerschain and @Mottie will sync up on the H1/Breadcrumb content

**Comment by Mottie** [comment]
I opened a pull request with these updates, but I still have some questions that need to be addressed before merging:
- Julie's last comment in Figma says we should keep the H1 & H2 the same and change the H3 of the page?
- If we change the H1:
  - When we navigate into the 4142 pages, the focus goes to the H3 and scrolls it to the top of the browser window, so the page title and subtitle aren't visible unless you scroll up. What do we do?
  - Within the 4142 flow, should we also update the breadcrumbs to match the H1? Skyler doesn't think so, but in the design we're only seeing the collapsed breadcrumb, not the full breadcrumb.
- What do we do about the subtitles? Julie suggested to update the subtitle to include both the main & support form. And do we change it based on the page?
- The introduction page will have 2 OMB info sections and both with the "View Privacy Act Statement" button - I haven't gotten a response from the Design System Team about hiding the modal button yet - so, should we continue to pursue this and should we? Do forms 4142 and 995 have the same privacy act statement?
- The alert that appears when you unselect the non-VA medical record authorization includes this text: "Or, go back a page and select No where we ask about private medical records." Does this need to be changed to "non-VA medical records"?

**Comment by anniebtran** [comment]
Still waiting on some feedback about H1s and subtitles, etc.

**Comment by eileen-coforma** [comment]
Approval from Julie to move on updates. FIGMA link has been updated.

**Comment by Mottie** [comment]
Work merged in and ready for review in staging

**Comment by eileen-coforma** [comment]
@Mottie so fast. A few minor things

<img width="618" alt="Screenshot 2024-06-27 at 11 16 07 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/111457818/455ddf59-cc3f-49ab-9947-6d7fbbd89473">

This should read VA Form 21-4142 and 21-4142a

<img width="567" alt="Screenshot 2024-06-27 at 11 18 06 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/111457818/59690cb5-53a5-4f6f-9a44-adb21d7bc46f">

H3 should read **Authorize the release of non-VA medical records to the VA**

**Comment by Mottie** [comment]
Fix has been merged in and ready for re-review

**Comment by jstrothman** [comment]
@Mottie When I click `Continue` on
https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/supporting-evidence/va-medical-records?index=0
the h3 "Would you like us to request your non-VA..." has a black outline, and I would have expected it to be yellow.
![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/1080991/f98d79f8-627a-4c71-9468-07029680f913)

**Comment by jstrothman** [comment]
@Mottie @eileen-coforma On the [intro page](https://staging.va.gov/decision-reviews/supplemental-claim/file-supplemental-claim-form-20-0995/introduction), the OMB info for 4142 has an expiration date of: ` 7/31/2024`

At first I thought we should change the OMB info, as the pdf form we're generating has the previous expiration date. However, I see in [reginfo.gov](https://www.reginfo.gov/public/Forward?SearchTarget=PRA&textfield=2900-0858&Image61.x=5&Image61.y=11) that the current expiration date of `7/31/2024` was assigned as an approval without any change to the form. 

We should still update the form we generate, as [described in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/86760) -- and should make sure to get the next version of the form, as the currently published one expires in less than 30 days.

**Comment by Mottie** [comment]
> h3 "Would you like us to request your non-VA..." has a black outline, and I would have expected it to be yellow.

This is a known issue and has been reported to the DST
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2521
- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2936
